tests/robottelo/test_vm.py

```diff
@@ -14,9 +14,10 @@ class VirtualMachineTestCase(unittest.TestCase):
     def setUp(self):
         super(VirtualMachineTestCase, self).setUp()
         self.properties_backup = conf.properties.copy()
+        conf.properties['clients.provisioning_server'] = None
```

Comment:
We're doing backup and restore before and after tests accordingly. The only thing that was missing - clearing provisioning server for testing purposes.